engine/src/flutter/impeller/display_list/aiks_dl_basic_unittests.cc

```diff
@@ -35,6 +35,76 @@ TEST_P(AiksTest, CanRenderColoredRect) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+namespace {
+using DrawRectProc =
+    std::function<void(DisplayListBuilder&, const DlRect&, const DlPaint&)>;
+
+[[maybe_unused]]
```

Comment:
How maybe_unused?

Reply:
Oh, crap, that was in there for testing. Thanks for finding that.